networking/src/APIClients/user/UserAPIClient.ts

```diff
@@ -0,0 +1,129 @@
+import { BaseAPIClient } from '../BaseAPIClient';
+import { IUserAPIClient } from './interfaces/IUserAPIClient';
+
+import {
+  CreateUserInput,
+  CreateUserMutation,
+  CreateUserMutationVariables,
+  DeleteUserInput,
+  DeleteUserMutation,
+  DeleteUserMutationVariables,
+  TeacherIdAuthInput,
+  TeacherIdAuthMutation,
+  TeacherIdAuthMutationVariables,
+  UpdateUserInput,
+  UpdateUserMutation,
+  UpdateUserMutationVariables
+
+} from "../../AWSMobileApi";
+
+import {
+  createUser,
+  deleteUser,
+  updateUser, 
+  teacherIdAuth
+} from "../../graphql";
+
+// interface IUser {
+//   id?: string | null,
+//   userName: string,
+//   title?: string | null,
+//   firstName?: string | null,
+//   lastName?: string | null,
+//   email: string,
+//   password?: string | null,
+//   gamesMade?: number | null,
+//   questionsMade?: number | null,
+// }
+
+export class UserAPIClient
+  extends BaseAPIClient
+  implements IUserAPIClient
+{
+  async createUser<IUser>( 
+    createUserInput: CreateUserInput
+  ): Promise<IUser> {
```

Comment:
you can write this comment as a JSDOC comment block. Here's some info on the formatting:
https://www.jetbrains.com/help/webstorm/creating-jsdoc-comments.html#ws_js_jsdoc_typescript_syntax

(I usually only use @params and @returns)